lib/collaboration/config/config.go

```diff
@@ -166,13 +166,13 @@ func (c *CollaborationConfig) parseTransformationSpec(path string) (*Transformat
 		return nil, fmt.Errorf("unable to unmarshal to TransformationGroupSpec %s", transformationYamlPath)
 	}
 	for i, tSpec := range tResult.Transformations {
-		tResult.Transformations[i].AppLocation = filepath.Join(path, tSpec.AppLocation)
+		tResult.Transformations[i].AppLocation = resolveRelativeAddress(path, tSpec.AppLocation)
```

Comment:
relative addresses were not supposed to be done for actual unix filepaths but internal graph node references.